tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -365,6 +365,458 @@ Synthetics:
+
+Summary:
+Schema => SemanticDB v3
+Uri => semanticdb/integration/src/main/scala/example/Types.scala
+Text => non-empty
+Language => Scala212
+Symbols => 100 entries
+Occurrences => 195 entries
+Diagnostics => 0 entries
+Synthetics => 0 entries
+
+Symbols:
+_root_.scala. => package scala
+_root_.scala.Int# => abstract final class Int.{+111 decls}
+  extends AnyVal
+_root_.scala.Predef.`???`()Lscala/Nothing;. => def ???: : Nothing
+  Nothing => _root_.scala.Nothing#
+_root_.scala.annotation. => package annotation
+_root_.scala.annotation.StaticAnnotation# => trait StaticAnnotation
+_root_.scala.language. => final object language
+_root_.scala.language.existentials. => implicit lazy val existentials: : existentials
+  existentials => _root_.scala.languageFeature.existentials#
+_root_.scala.language.higherKinds. => implicit lazy val higherKinds: : higherKinds
+  higherKinds => _root_.scala.languageFeature.higherKinds#
+_root_.scala.package.List# => val List: : List.type
+  List => _root_.scala.collection.immutable.List.
+_root_.types. => package types.{+8 members}
+_root_.types.B# => class B
+_root_.types.B#`<init>`()V. => primaryctor <init>: (): B
+  B => _root_.types.B#
+_root_.types.C# => class C.{+1 decls}
+  extends AnyRef
+_root_.types.C#`<init>`()V. => primaryctor <init>: (): C
+  C => _root_.types.C#
+_root_.types.P# => class P.{+5 decls}
+  extends AnyRef
+_root_.types.P#C# => class C.{+1 decls}
+  extends AnyRef
+_root_.types.P#C#`<init>`()V. => primaryctor <init>: (): P.this.type.C
+  P => _root_.types.P#
+  C => _root_.types.P#C#
+_root_.types.P#X# => class X.{+1 decls}
+  extends AnyRef
+_root_.types.P#X#`<init>`()V. => primaryctor <init>: (): P.this.type.X
+  P => _root_.types.P#
+  X => _root_.types.P#X#
+_root_.types.P#`<init>`()V. => primaryctor <init>: (): P
+  P => _root_.types.P#
+_root_.types.P#x. => val x: : P.this.type.X
+  P => _root_.types.P#
+  X => _root_.types.P#X#
+_root_.types.T# => class T.{+5 decls}
+  extends AnyRef
+_root_.types.T#C# => class C.{+1 decls}
+  extends AnyRef
+_root_.types.T#C#`<init>`()V. => primaryctor <init>: (): T.this.type.C
+  T => _root_.types.T#
+  C => _root_.types.T#C#
+_root_.types.T#X# => class X.{+1 decls}
+  extends AnyRef
+_root_.types.T#X#`<init>`()V. => primaryctor <init>: (): T.this.type.X
+  T => _root_.types.T#
+  X => _root_.types.T#X#
+_root_.types.T#`<init>`()V. => primaryctor <init>: (): T
+  T => _root_.types.T#
+_root_.types.T#x. => val x: : T.this.type.X
+  T => _root_.types.T#
+  X => _root_.types.T#X#
+_root_.types.Test. => final object Test
+_root_.types.Test.C# => class C.{+45 decls}
+  extends M
+_root_.types.Test.C#$anon#k()I. => abstract def k: : Int
+  Int => _root_.scala.Int#
+_root_.types.Test.C#$anon#λ# => type λ: [T>: Nothing<: Any] => List[T]
+  T => _root_.types.Test.C#$anon#λ#[T]
+  List => _root_.scala.package.List#
+  T => _root_.types.Test.C#$anon#λ#[T]
+_root_.types.Test.C#$anon#λ#[T] => typeparam T: >: Nothing<: Any
```

Comment:
What happens for two similar symbols in the same scope? Do the symbols conflict?

Reply:
There cannot be two symbols with the same id. If this happens, this means it's a bug.

Reply:
T is printed twice above because it occurs twice in the type signature of `_root_.types.Test.C#$anon#λ#`, i.e.: `type λ: [T >: Nothing <: Any] => List[T]`.